wazo_acceptance/helpers/incall.py

```diff
@@ -17,3 +17,19 @@ def add_extension(self, incall, extension):
         with self._context.helpers.bus.wait_for_asterisk_reload(dialplan=True):
             self._confd_client.incalls(incall).add_extension(extension)
         self._context.add_cleanup(self._confd_client.incalls(incall).remove_extension, extension)
+
+    def add_schedule(self, incall, schedule):
+        with self._context.helpers.bus.wait_for_asterisk_reload(dialplan=True):
+            self._confd_client.incalls(incall).add_schedule(schedule)
+        self._context.add_cleanup(self._confd_client.incalls(incall).remove_schedule, schedule)
```

Comment:
This resource do not have the constraint to be dissociated before removing. So You can remove the helper and use the client directly in the step

Reply:
I will have to cleanup it in the steps anyways to keep the env clean.
I think it's more clean to have all helpers doing set/cleanup and if people don't want the automated cleanup for some reason it have the explicitly write it, no ?